gitops/core/base/crds/traefik/crds.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta1
+kind: GitRepository
+metadata:
+  name: traefik-crd-source
+  namespace: flux-system
+spec:
+  interval: 10m
+  url: https://github.com/traefik/traefik-helm-chart.git
+  ref:
+    semver: ">=9.14.2 <9.15.0"
+  ignore: |
+    # exclude all
+    /*
+    # include deploy crds dir
+    !/traefik/crds/
+---
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta1
+kind: Kustomization
+metadata:
+  name: traefik-crd
+  namespace: flux-system
+spec:
+  interval: 5m
+  prune: false
+  sourceRef:
+    kind: GitRepository
+    name: traefik-crd-source
+  healthChecks:
+    - apiVersion: helm.toolkit.fluxcd.io/v1beta1
+      kind: HelmRelease
+      name: traefik
+      namespace: traefik
```

Comment:
AFAIK healthchecks are supports to be for the resource the kustomize deploys and manages, not necessarily for a resource created in another kustomize, similar to how healthchecks work in deployments/pods

Reply:
Thanks for this. Before merging i'm going to read up on the healthchecks but shouldn't be needed